src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -11,8 +11,11 @@
 #include "preferences/dialog/dlgpreflibrary.h"
 #include "library/dlgtrackmetadataexport.h"
 #include "sources/soundsourceproxy.h"
+#include "widget/wsearchlineedit.h"
 
-#define MIXXX_ADDONS_URL "http://www.mixxx.org/wiki/doku.php/add-ons"
```

Comment:
This macro is still in use. Is this an unrelated change, slipped in?

Reply:
Slipped in. I didn't notice, because I'm already using a version without plugins for development. I've removed this obsolete #define now from master to prevent such mistakes in the future.